Automattic/wp-calypso
# Remove the domain privacy protection selector in the checkout

#### Changes proposed in this Pull Request

* as discussed we want to remove the option to select domain privacy when purchasing a new domain as it is now provided for free and people can always enable/disable the privacy protection at any time after domain purchase. We believe that we can provide privacy for all new domain registrations.

see full discussion here - p99Zz8-Ql-p2

#### Testing instructions

* add a new domain in your cart and proceed to the next step where you enter your contact information - there should be no option to enable/disable privacy
